src/store/features/templates/reducer.ts

```diff
@@ -64,3 +64,12 @@ export const templatesReducer = createReducer(initialState, (builder) => {
       return newState;
     });
 });
+
+export const recommendedTemplateColumns: Record<string, ImportReducedTemplateWithColumns["columns"]> = (recommendedTemplatesJson as ImportReducedTemplateWithColumns[]).reduce(
+  (acc, tpl, idx) => {
+    const id = `recommended-${idx}`;
+    acc[id] = tpl.columns;
+    return acc;
+  },
+  {} as Record<string, ImportReducedTemplateWithColumns["columns"]>
+);
```

Comment:
I don't understand why you're constructing an object out of the array here, wouldn't an array be easier or was there a specific reason?